package com.java.oops.exception_handling;

public class TryWIthFinally {

	public static void main(String[] args) {


		try {
			System.out.println(9/0);
		} finally {
			
		}
	

	}

}
